.env

```diff
@@ -0,0 +1,3 @@
+LINK_DB_USER=link-db-user
+LINK_DB_PASS=link-db-pass
+REDIS_PASS=suT1ChwYtkwWfbV1bUfRvQZO1YNhNRMj
```

Comment:
Is this sensitive?

Reply:
No. This is a new uniquely generated value just for this docker-compose.